pkg/binlogfile/binlogger_test.go

```diff
@@ -214,16 +215,31 @@ func (s *testBinloggerSuite) TestGC(c *C) {
 	b, ok := bl.(*binlogger)
 	c.Assert(ok, IsTrue)
 	err = b.rotate()
+	// 2. rotate creates a new binlog file with index 1
 	c.Assert(err, IsNil)
 
+	// No binlog files should be collected,
+	// because both of the files has an index that's >= 0
 	time.Sleep(10 * time.Millisecond)
-	b.GC(time.Millisecond, binlog.Pos{})
+	b.GC(time.Millisecond, binlog.Pos{Suffix: 0})
 
 	names, err := ReadBinlogNames(b.dir)
 	c.Assert(err, IsNil)
 	c.Assert(names, HasLen, 2)
-	c.Assert(names[0], Equals, BinlogName(0))
-	c.Assert(names[1], Equals, BinlogName(1))
+	for i, name := range names {
+		suffix, _, err := ParseBinlogName(name)
+		c.Assert(err, IsNil)
+		c.Assert(suffix, Equals, uint64(i))
+	}
+
+	// The one with index 0 should be collected
```

Comment:
do you means the one with index 0 should be gc?

Reply:
Fixed.